GameServer/Logic/Chatbox/Commands/ChCommand.cs

```diff
@@ -34,7 +34,7 @@ public override void Execute(Peer peer, bool hasReceivedArguments, string argume
                 _playerManager.GetPeerInfo(peer).Champion.Y
             );
             c.Model = split[1]; // trigger the "modelUpdate" proc
-            c.SetTeam(_playerManager.GetPeerInfo(peer).Champion.Team);
```

Comment:
I'd assume setting the team will need some special logic besides just changing the value of the variable, so this should remain a function (`SetTeam` as a setter function, but a (publicly read only) property `Team` should exist for storing the value)